### PR TITLE
Subgraph isomorphism: API aligning - task added

### PR DESCRIPTION
**Pull request by orazve**

### CODE REVIEW

File: cpp/oneapi/dal/algo/subgraph_isomorphism/test/badargs.cpp [hunk 1]
```diff
@@ -186,15 +186,6 @@ SUBGRAPH_ISOMORPHISM_BADARG_TEST("Throws if match count is negative") {
         invalid_argument);
 }
 
-SUBGRAPH_ISOMORPHISM_BADARG_TEST("Throws if match count is positive") {
```
Comment (orrrrtem) [inline cpp/oneapi/dal/algo/subgraph_isomorphism/test/badargs.cpp:189]: Just for a check. Do you really want to delete this?

**Review by orrrrtem — APPROVED** [review]

File: cpp/oneapi/dal/algo/subgraph_isomorphism/test/badargs.cpp (hunk 1 above)
Comment (orazve) [reply cpp/oneapi/dal/algo/subgraph_isomorphism/test/badargs.cpp]: Reverted, thank you/

**Pull request merged**